packages/node_modules/@webex/plugin-meetings/src/locus-info/infoUtils.js

```diff
@@ -24,7 +24,7 @@ InfoUtils.parseMeetingType = (info) => {
   let type = UNKNOWN;
   // Parse all the value and get the type of meeting
   // 1:1 meeting
-  if (info.locusTags && info.locusTags.indexOf(ONE_ON_ONE) >= 0) {
+  if (info.locusTags && info.locusTags.indexOf(ONE_ON_ONE) >= 0 || info.maxParticipants === 3) {
```

Comment:
hmmmmm, what about if both users are joined with devices???

Reply:
Added that to the TODO list . Need to test it

Reply:
okay, we should probably test before merging on this one i think

Reply:
tested

Reply:
What is this check doing? Seems strange to assert that we have exactly 3 participants.